src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -64,6 +65,9 @@
     @FXML
     private StackPane listPanelPlaceholder;
 
+    @FXML
+    private StackPane detailsPanelPlaceholder;
```

Comment:
might need to rename to something like left and right panel placeholder because Dashboard will also be using these components